wifiphisher.py

```diff
@@ -869,6 +869,16 @@ def get_hostapd():
         else:
             sys.exit(('[' + R + '-' + W + '] hostapd' +
                      'not found in /usr/sbin/hostapd'))
+    if not os.path.isfile('/usr/sbin/hostapd'):
+        sys.exit((
+            '\n[' + R + '-' + W + '] Unable to install the \'hostapd\' package!\n' +
+            '[' + T + '*' + W + '] This process requires a persistent internet connexion!\n' +
+            'Please follow the link below to configure your sources.list\n' +
+            B + 'http://docs.kali.org/general-use/kali-linux-sources-list-repositories\n' + W +
+            '[' + G + ' + ' + W + '] Run apt-get update for changes to take effect.\n' +
+            '[' + G + ' + ' + W + '] Rerun the script again to install hostapd.\n' +
```

Comment:
I think it's better if you remove the spaces that surround the "+" symbol to be similar with the rest of prefixes.